libs/blocks/modal/modal.css

```diff
@@ -195,3 +191,7 @@
     height: 845px;
   }
 }
+
+body.disable-scroll {
```

Comment:
Issues reported for commerce modal are getting fixed by using css class name without the tag name i.e

.disable-scroll @saragajic11 could you please take a look. Thanks

Reply:
Hi @Ruchika4 I have update the modal.css file (https://github.com/adobecom/milo/pull/1457/commits/bd85eb224bdd7c54380709f1815fecc1066ed36b) per your suggestion but I can still see that commerce modal is not opening the way it is on main branch...

Reply:
oh ok, could you please investigate then what is causing the issue. As with current code it is working as expected and it's breaking with this PR. Thanks